app/src/main/res/xml/video_audio_settings.xml

```diff
@@ -146,5 +146,14 @@
             android:key="@string/use_inexact_seek_key"
             android:summary="@string/use_inexact_seek_summary"
             android:title="@string/use_inexact_seek_title"/>
+
+        <ListPreference
+          app:iconSpaceReserved="false"
+          android:defaultValue="@string/seek_duration_default_key"
+          android:entries="@array/seek_duration_description"
+          android:entryValues="@array/seek_duration_value"
+          android:key="@string/seek_duration_key"
+          android:summary="%s"
```

Comment:
Are you sure this is ok? According to [this thread](https://stackoverflow.com/questions/7017082/change-the-summary-of-a-listpreference-with-the-new-value-android) the implementation is buggy

Reply:
To be honest, i think i just copied it from the resolution preference [here](https://github.com/TeamNewPipe/NewPipe/blob/915455d4e5023083e2e2855c52abea17982c291d/app/src/main/res/xml/video_audio_settings.xml#L13). 
It seems to at least work on my API 25 and 29 devices and a 21 emulator.

Reply:
Oh ok, thanks ;-)